lib/js/dom.nim

```diff
@@ -1467,6 +1467,7 @@ else:
   proc insertBefore*(n, newNode, before: Node) {.importcpp.}
   proc getElementById*(d: Document, id: cstring): Element {.importcpp.}
   proc createElement*(d: Document, identifier: cstring): Element {.importcpp.}
+  proc createElementNS*(d: Document, namespaceURI, qualifiedIdentifier: cstring): Element {.importcpp.}
```

Comment:
```suggestion
  proc createElementNS*(d: Document, namespaceURI, qualifiedIdentifier: cstring): Element {.importcpp.}
    ## https://developer.mozilla.org/en-US/docs/Web/API/Document/createElementNS
```

we should start adding doc refs everywhere IMO; it's more user friendly (and new js modules do that)

Reply:
Ideally someone should look into https://github.com/nim-lang/Nim/issues/11021